src/reference/asciidoc/aggregator.adoc

```diff
@@ -157,17 +137,16 @@ public interface ReleaseStrategy {
 
 }
 ----
+====
 
-In general, any POJO can implement the completion decision logic if it provides a method that accepts a single `java.util.List` as an argument (parameterized lists are supported as well), and returns a boolean value.
-This method will be invoked after the arrival of each new message, to decide whether the group is complete or not, as follows:
-
-* if the argument is a `java.util.List<T>`, and the parameter type T is assignable to `Message`, then the whole list of messages accumulated in the group will be sent to the method
+In general, any POJO can implement the completion decision logic if it provides a method that accepts a single `java.util.List` as an argument (parameterized lists are supported as well) and returns a boolean value.
+This method is invoked after the arrival of each new message, to decide whether the group is complete or not, as follows:
 
-* if the argument is a non-parametrized `java.util.List` or the parameter type is not assignable to `Message`, then the method will receive the payloads of the accumulated messages
+* If the argument is a `java.util.List<T>`, and the parameter type `T` is assignable to `Message`, the whole list of messages accumulated in the group is sent to the method.
+* If the argument is a non-parametrized `java.util.List` or the parameter type is not assignable to `Message`, the method receives the payloads of the accumulated messages.
+* The method must return `true` if the message group is ready for aggregation or false otherwise.
 
-* the method must return true if the message group is ready for aggregation, and false otherwise.
-
-For example:
+The following example shows how to use the `@ReleaseStrategy` annotation for a `List` of type `Message`:
 
 [source,java]
```

Comment:
This code snippet is missed to be wrapped with the `====`.
Or am I missing the point why and when should we wrap them?

Thanks